wallet/src/wallet/mod.rs

```diff
@@ -97,6 +108,7 @@ pub struct Wallet<B: storage::Backend> {
     db: Store<B>,
     key_chain: MasterKeyChain,
     accounts: BTreeMap<U31, Account>,
+    unsynced_account: Option<(U31, Account)>,
```

Comment:
Please add documentation explaining the logic behind this

Reply:
Looking more into the code, why is this an `Option<(U31, Account)>`, and not a map so that multiple accounts can be synced and created? Is there a reason beyond that this is easier to do, in a way?

Reply:
it just fallows the logic that only 1 account can be empty, so you can't create multiple unsynced accounts.